.github/workflows/fix.coverity-2024.12.patch

```diff
@@ -0,0 +1,341 @@
+diff -Naur config.orig/templates/clang/llvm/clang_llvm_switches.dat config/templates/clang/llvm/clang_llvm_switches.dat
```

Comment:
I haven't had a good idea about where we should put this file.

We have `.github/scripts`, but I don't feel like it belongs there. Any ideas?